### ISSUE TITLE
🔗 ⛔ 🛟 Support Request: I can't use the mega Menu widget after updating (Duplicate of #23790)

### ISSUE THREAD

**Issue by zf-assorttech** [elementor/elementor#23861]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

### Description

when i update my elementor to 3.16 dev version 4 my mega menu cant work 

### Steps to reproduce

once I update my Elementor to the latest version my mega menu template is not shown on hover even on click 
when I downgrade my Elementor version to 3.15 my mega menu runs perfectly

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt



== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server - (GPL) v5.7.42
	PHP Version: 8.1.22
	PHP Memory Limit: 368M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://assorttech.com
	Home URL: https://assorttech.com
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 40M
	Max Memory limit: 368M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Elementor
		Version: 3.15.3
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.15.1
		Author: Elementor.com

	Jetpack Protect
		Version: 1.4.1
		Author: Automattic - Jetpack Security team

	Limit Login Attempts Reloaded
		Version: 2.25.25
		Author: Limit Login Attempts Reloaded

	Site Kit by Google
		Version: 1.109.0
		Author: Google

	Slider Revolution
		Version: 6.6.11
		Author: ThemePunch

	Wordfence Security
		Version: 7.10.3
		Author: Wordfence

	WP Revisions Control
		Version: 1.4.3
		Author: Erick Hitter

	Yoast SEO
		Version: 21.2
		Author: Team Yoast


== Must-Use Plugins ==
	WordPress automation by Installatron
		Version: 
		Author: 


== Elements Usage ==
	
	container : 5
		button : 100
		container : 467
		counter : 60
		heading : 272
		nested-tabs : 25
		spacer : 2
		text-editor : 211
	archive : 3
		archive-posts : 1
		container : 4
		loop-carousel : 1
		loop-grid : 1
		nav-menu : 1
		search-form : 1
	footer : 1
		breadcrumbs : 1
		container : 12
		heading : 9
		html : 1
		icon-box : 1
		icon-list : 1
		nav-menu : 4
		slider_revolution : 1
		text-editor : 1
	header : 3
		button : 5
		container : 21
		heading : 9
		icon-box : 6
		mega-menu : 1
		nav-menu : 1
		progress-tracker : 1
		theme-site-logo : 2
	page : 6
		button : 1
		container : 12
		countdown : 1
		divider : 1
		heading : 11
		icon-box : 18
		image : 2
	popup : 1
		icon-box : 6
	wp-page : 19
		breadcrumbs : 14
		button : 91
		container : 720
		counter : 36
		divider : 18
		google_maps : 3
		heading : 619
		html : 2
		icon : 36
		icon-box : 133
		icon-list : 33
		image : 43
		image-box : 3
		image-carousel : 1
		loop-carousel : 1
		loop-grid : 23
		lottie : 2
		nested-tabs : 20
		posts : 1
		shortcode : 2
		slider_revolution : 8
		social-icons : 14
		spacer : 5
		template : 2
		text-editor : 232
		theme-page-title : 4
		toggle : 6
	single-post : 4
		breadcrumbs : 1
		container : 32
		facebook-page : 1
		gallery : 1
		heading : 10
		loop-grid : 2
		post-info : 7
		post-navigation : 2
		share-buttons : 1
		slider_revolution : 2
		social-icons : 1
		table-of-contents : 1
		theme-post-content : 4
		theme-post-excerpt : 2
		theme-post-featured-image : 1
		theme-post-title : 3
		toggle : 1
	single-page : 1
		nav-menu : 1
		post-info : 4
		search-form : 1
		spacer : 2
		theme-page-title : 1
		theme-post-excerpt : 2
		theme-post-featured-image : 2
		theme-post-title : 2
	loop-item : 5
		breadcrumbs : 1
		button : 3
		container : 16
		heading : 3
		post-info : 8
		theme-post-excerpt : 2
		theme-post-featured-image : 5
		theme-post-title : 6
	error-404 : 1
		button : 1
		container : 1
		image : 1


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap
	load_fa4_shim: yes


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha: Active


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Active
	Hello Theme Header &amp; Footer: Active
	Editor Top Bar: Active
	Landing Pages: Active
	Nested Elements: Active
	Lazy Load Background Images: Active
	Global Style Guide: Active
	Page Transitions: Active
	Notes: Active
	Loop: Active
	Form Submissions: Active
	Scroll Snap: Active
	Menu: Active
	Taxonomy Filter: Active


== Log ==
	
Log: showing 20 of 312023-09-12 05:28:50 [info] Elementor/Upgrades - _v_3_16_0_container_updates Finished 
2023-09-12 05:28:50 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-09-12 05:28:50 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-09-12 05:28:50 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.0-dev3',
  'to' => '3.16.0-dev4',
)]
2023-09-12 06:35:08 [info] Elementor custom task(s) process has been queued. [array (
  0 => 
  array (
    0 => 
    array (
      0 => 'Elementor\Core\Upgrade\Custom_Tasks',
      1 => 'opt_in_recalculate_usage',
    ),
    1 => 
    array (
      0 => 'Elementor\Core\Upgrade\Custom_Tasks',
      1 => 'opt_in_send_tracking_data',
    ),
  ),
)]
2023-09-12 06:35:08 [info] elementor::custom_task_manger Started 
2023-09-12 06:35:08 [info] Elementor/Tasks - opt_in_recalculate_usage Start  
2023-09-12 06:35:09 [info] Elementor/Tasks - opt_in_recalculate_usage Finished 
2023-09-12 06:35:09 [info] Elementor/Tasks - opt_in_send_tracking_data Start  
2023-09-12 06:35:09 [info] Elementor/Tasks - opt_in_send_tracking_data Finished 
2023-09-12 06:35:09 [info] elementor::custom_task_manger Completed 
2023-09-12 11:08:17 [info] elementor::elementor_updater Started 
2023-09-12 11:08:17 [info] Elementor/Upgrades - _on_each_version Start  
2023-09-12 11:08:17 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.0-dev4',
  'to' => '3.16.0',
)]
2023-09-12 11:08:18 [info] Elementor/Upgrades - _on_each_version Finished 
2023-09-12 11:08:18 [info] Elementor/Upgrades - _v_3_16_0_container_updates Start  
2023-09-12 11:08:19 [info] Elementor/Upgrades - _v_3_16_0_container_updates Finished 
2023-09-12 11:08:19 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-09-12 11:08:19 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-09-12 11:08:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.0-dev4',
  'to' => '3.16.0',
)]

PHP: showing 8 of 8PHP: 2023-09-08 06:30:58 [warning X 11][../wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key &quot;data&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-08 06:30:58 [warning X 48][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-09-08 10:16:07 [notice X 8][../wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-12 05:35:29 [warning X 103][../wp-content/plugins/elementor/includes/widgets/image-carousel.php::1021] Attempt to read property &quot;post_excerpt&quot; on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-12 05:43:13 [warning X 1][../wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-13 06:48:08 [warning X 222][../wp-content/plugins/elementor/includes/widgets/image-carousel.php::1022] Attempt to read property &quot;post_excerpt&quot; on null [array (
  'trace' => '
#0: ../wp-content/plugins/elementor/includes/widgets/image-carousel.php(1022): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/includes/widgets/image-carousel.php(884): Elementor\Widget_Image_Carousel -> get_image_caption()
#2: ../wp-content/plugins/elementor/includes/base/controls-stack.php(2243): Elementor\Widget_Image_Carousel -> render()
#3: ../wp-content/plugins/elementor/includes/base/widget-base.php(609): Elementor\Controls_Stack -> render_by_mode()
#4: ../wp-content/plugins/elementor/includes/base/widget-base.php(750): Elementor\Widget_Base -> render_content()
',
)]
PHP: 2023-09-13 08:30:52 [warning X 5][../wp-content/plugins/elementor/core/page-assets/data-managers/font-icon-svg/font-awesome.php::50] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-13 10:54:23 [warning X 3][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::171] Undefined array key &quot;editor_post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 3 of 3JS: 2023-09-13 06:57:17 [error X 2][https://assorttech.com/wp-includes/js/jquery/jquery.min.js?ver=3.6.4:2:31823] Cannot read properties of undefined (reading &#039;value&#039;) 
JS: 2023-09-14 06:01:56 [error X 4][https://assorttech.com/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.15.1:2:64105] Cannot convert undefined or null to object 
JS: 2023-09-14 07:16:25 [error X 9][https://assorttech.com/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.15.3:2:6655] Cannot read properties of undefined (reading &#039;autoplay&#039;) 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	



````

</details>

**Comment by Daveden2** [comment]
A couple of points
1. I hope you are NOT using the dev version on a live site
2.  I'd suggest that you **deactivate Improved CSS Loading** by going to _Elementor > Settings > Features_ . 
3. Then **Clear your cache** and Regenerate CSS by going to _Elementor > Tools > Regerate Files_

**Comment by nataib** [comment]
I have the exact same issue - when I update to elementor 3.16, my mega menu shows on hover, but when I try to mouse over the mega menu it disappears.

I've followed all the instructions here: https://github.com/elementor/elementor/issues/23753

I've disabled server caching, deactivated optimization & caching wordpress plugin, deactivated Improved CSS Loading, Regenerated CSS in Elementor and cleared my browser cache - and the mega menu still doesn't work.

I rolled back to version 3.15 and it works again.
